lib/settings_base.py

```diff
@@ -229,6 +229,11 @@ def lazy_langs(languages):
 # Tarballs in DUMPED_USERS_PATH deleted 30 days after they have been written.
 DUMPED_USERS_DAYS_DELETE = 3600 * 24 * 30
 
+# path that isn't just one / deep, and doesn't require any locale or app.
+SUPPORTED_NONAPPS_NONLOCALES_DEEP = (
+    'api/v3'
```

Comment:
So that's just a simple string, not a list or tuple of strings? If so, get rid of the parens. If not, add a comma.